src/components/Header/Header.tsx

```diff
@@ -70,8 +59,8 @@ function Header() {
         <div className={classes.main__bottom}>
           <ul className={classes.main__dropdowns}>
             {dropdownsHeaders.map((el) => (

```

Comment:
you can use destructuring if you do not have many elements. dropdownsHeaders.map(({ header, options }) =>